spec/helpers/bookmarks_helper_spec.rb

```diff
@@ -0,0 +1,223 @@
+require "spec_helper"
+
+describe BookmarksHelper do
+  let(:bookmarker) { create(:user) }
+  let(:external_work_bookmark) { create(:external_work_bookmark, pseud: bookmarker.default_pseud) }
+  let(:series_bookmark) { create(:series_bookmark, pseud: bookmarker.default_pseud) }
+  let(:work_bookmark) { create(:bookmark, pseud: bookmarker.default_pseud) }
+  let(:external_work) { external_work_bookmark.bookmarkable }
+  let(:series) { series_bookmark.bookmarkable }
+  let(:series_creator) { series.users.first }
+  let(:work) { work_bookmark.bookmarkable }
+  let(:work_creator) { work.users.first }
+
+  describe "#css_classes_for_bookmark_blurb" do
+    let(:default_classes) { "bookmark blurb group" }
+
+    context "when bookmarkable is ExternalWork" do
+      it "returns string with default classes, creation info, and bookmarker info" do
+        result = helper.css_classes_for_bookmark_blurb(external_work_bookmark)
+        expect(result).to eq("#{default_classes} external-work-#{external_work.id} user-#{bookmarker.id}")
+      end
+
+      context "when bookmark is updated" do
```

Comment:
I didn't test updating the external work because that's... not really a thing we do, and it definitely wouldn't be updated in a way that would change the expected classes. I can add it if we want, though.

Reply:
Let's skip that.